### PR TITLE
Add docker support for e2e test dependency

### PR DESCRIPTION
**Pull request by sadlil**
@aerokite after this is done you can use this for e2e testing. please use the docker cmds in after `docker --version`

### CODE REVIEW

**Comment by mirshahriar** [comment]
What should be the IP address of docker? @sadlil

**Review by mirshahriar — DISMISSED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ before_script:
   - ./hack/scripts/protoc.sh
   - make tools
   - docker --version
+  - docker run -d -p 127.0.0.1:5432:5432 postgres:latest
```
Comment (sadlil) [inline .travis.yml]: You don't need to put `-p 127.0.0.1` just do `-p 5432:5432`

File: .travis.yml [hunk 2]
```diff
@@ -2,18 +2,28 @@ language: go
...
+services:
+  - docker
+
 matrix:
   include:
     - go: 1.10.1
 
+before_install:
+  - docker pull postgres:latest
+  - docker run -d -p 127.0.0.1:15432:5432 postgres:latest
```
Comment (sadlil) [inline .travis.yml]: use `-p 5432:5432` only. no need to put ip.

Comment (mirshahriar) [reply .travis.yml]: `5432` is occupied by another process. Can't use that.

File: .travis.yml [hunk 3]
```diff
@@ -11,8 +11,9 @@ matrix:
 
 before_install:
   - docker pull postgres:latest
-  - docker run -d -p 127.0.0.1:15432:5432 postgres:latest
-  - docker ps -a
+  - docker run -d -p 15432:5432 postgres:latest
+  - docker pull mysql:5.7
+  - docker run -d -p 13306:3306 -e MYSQL_ROOT_PASSWORD=secret -e MYSQL_DATABASE=test mysql:5.7
```
Comment (sadlil) [inline .travis.yml]: Can you check one thing. I hear travis already have mysql and posgres installed and running on build already.

**Comment by sadlil** [comment]
making this absolute.

**Pull request closed without merging**